src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/client/oasysApi/OasysApiClient.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.client.oasysApi
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.client.WebClient
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.client.BaseHMPPSClient
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.client.oasysApi.model.OasysAssessmentTimeline
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.client.oasysApi.model.OasysOffenceDetailWrapper
+
+@Component
+class OasysApiClient(
+  @Qualifier("oasysApiWebClient")
+  webClient: WebClient,
+) : BaseHMPPSClient(webClient, jacksonObjectMapper()) {
+
+  fun getAssessments(prisonerId: String) = getRequest<OasysAssessmentTimeline> {
```

Comment:
Can we call the parameter `prisonNumber` rather than `prisonerId`? That's the name of the field we get back on a person in prison, which we'd then use here. Would hopefully help clear up some confusion on `prisonerNumber`, `prisonNumber`, `nomisNumber` (all the same thing!) and `prisonId` (something else...)